src/main/java/tech/sourced/siva/Header.java

```diff
@@ -4,31 +4,54 @@
 import java.nio.file.attribute.PosixFilePermission;
 import java.util.Set;
 
+/**
+ * Set of properties an {@link IndexEntry} must have
```

Comment:
What do you think about `Header contains the properties of a single index entry in the siva file.`¿

Reply:
I'll change it to `Header contains the meta information from a file that a single {@link IndexEntry} keeps in the siva file.` because they aren't all the properties an `IndexEntry` has.